lib/qa/authorities/linked_data/rdf_helper.rb

```diff
@@ -0,0 +1,73 @@
+require 'rdf'
+require	'linkeddata'
+module Qa::Authorities
+  module LinkedData
+    module RdfHelper
+      private
+
+        def get_linked_data(url)
+          begin
+            graph = RDF::Graph.load(url)
+          rescue IOError => e
+            process_error(e, url)
+          end
+          graph
+        end
+
+        def process_error(e, url)
+          uri = URI(url)
+          response_code = ioerror_code(e)
+          case response_code
+          when '404'
+            raise Qa::TermNotFound, "#{uri} Not Found - Term may not exist at LOD Authority. (HTTPNotFound - 404)"
+          when '500'
+            raise Qa::ServiceUnavailable, "#{uri.hostname} on port #{uri.port} is not responding.  Try again later. (HTTPServerError - 500)"
+          when '503'
+            raise Qa::ServiceUnavailable, "#{uri.hostname} on port #{uri.port} is not responding.  Try again later. (HTTPServiceUnavailable - 503)"
+          else
+            raise Qa::ServiceUnavailable, "#{uri.hostname} on port #{uri.port} is not responding.  Try again later. (Cause - #{response_code})"
+          end
+        end
+
+        def ioerror_code(e)
+          msg = e.message
+          a = msg.size - 4
+          z = msg.size - 2
+          msg[a..z]
+        end
+
+        def filter_language(graph, language)
```

Comment:
Since this looks like the language filter is done on the client side rather than the remote, how hard would it be to allow it to accept a list of languages rather than a single one? Not a feature needed for right now but I often have cases that I need matches in one a set of potential English-style languages.